tasks/e2e/cypress/e2e/01-tutorial/sharedTests.js

```diff
@@ -213,38 +213,38 @@ export const test_routing_params = () =>
     cy.exec(`cd ${BASE_DIR}; yarn rw g cell BlogPost --force`)
     cy.exec(`cd ${BASE_DIR}; yarn rw g component BlogPost --force`)
 
-    cy.writeFile(path.join(BASE_DIR, 'web/src/Routes.js'), Step6_1_Routes)
+    cy.writeFile(path.join(BASE_DIR, 'web/src/Routes.jsx'), Step6_1_Routes)
     cy.writeFile(
-      path.join(BASE_DIR, 'web/src/pages/BlogPostPage/BlogPostPage.js'),
+      path.join(BASE_DIR, 'web/src/pages/BlogPostPage/BlogPostPage.jsx'),
       Step6_2_BlogPostPage
     )
     cy.writeFile(
-      path.join(BASE_DIR, 'web/src/components/BlogPostCell/BlogPostCell.js'),
+      path.join(BASE_DIR, 'web/src/components/BlogPostCell/BlogPostCell.jsx'),
       Step6_3_BlogPostCell
     )
     cy.writeFile(
       path.join(
         BASE_DIR,
-        'web/src/components/BlogPostCell/BlogPostCell.test.js'
+        'web/src/components/BlogPostCell/BlogPostCell.test.jsx'
       ),
       Step6_3_BlogPostCellTest
     )
     cy.writeFile(
-      path.join(BASE_DIR, 'web/src/components/BlogPost/BlogPost.js'),
+      path.join(BASE_DIR, 'web/src/components/BlogPost/BlogPost.jsx'),
       Step6_4_BlogPost
     )
     cy.writeFile(
-      path.join(BASE_DIR, 'web/src/components/BlogPost/BlogPost.test.js'),
+      path.join(BASE_DIR, 'web/src/components/BlogPost/BlogPost.test.jsx'),
       Step6_4_BlogPostTest
     )
     cy.writeFile(
-      path.join(BASE_DIR, 'web/src/components/BlogPostsCell/BlogPostsCell.js'),
+      path.join(BASE_DIR, 'web/src/components/BlogPostsCell/BlogPostsCell.jsx'),
       Step6_5_BlogPostsCell
     )
     cy.writeFile(
       path.join(
         BASE_DIR,
-        'web/src/components/BlogPostsCell/BlogPostsCell.mock.js'
+        'web/src/components/BlogPostsCell/BlogPostsCell.mock.jsx'
```

Comment:
I think this should be JS - mocks are just regular JS files with no JSX right?

Reply:
Yep, that one slipped through!